src/man/sssd.conf.5.xml

```diff
@@ -733,26 +733,24 @@
                             When SSSD switches to offline mode the amount of
                             time before it tries to go back online will
                             increase based upon the time spent disconnected.
-                            This value is in seconds and calculated by the
-                            following:
-                        </para>
-                        <para>
-                             offline_timeout + random_offset
-                        </para>
-                        <para>
-                            The random offset value is from 0 to 30.
+                            By default SSSD uses incremental behaviour to
+                            calculate delay in between retries.
+                            The longer backend is offline, the bigger delays
+                            in between attempts to checking it status again.
```

Comment:
This phrase sounds weird to me. I know what you mean but I don't think it makes sense in English. My proposal would be to change it for something similar to:
`So, the wait time for a given retry will be longer than the wait time for the previous one.`

Reply:
Fixed